app/controllers/comments_controller.rb

```diff
@@ -9,6 +9,12 @@ def create
     else
       flash[:alert] = t("comment.empty")
     end
+   
+    Comment.all.each do |comment|
```

Comment:
why are you using Comment.all here? That will pull up all the comments from the database